<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistic</class>
 <widget class="QWidget" name="Statistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>993</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1212</width>
        <height>991</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <property name="spacing">
        <number>32</number>
       </property>
       <property name="leftMargin">
        <number>64</number>
       </property>
       <property name="topMargin">
        <number>64</number>
       </property>
       <property name="rightMargin">
        <number>64</number>
       </property>
       <property name="bottomMargin">
        <number>64</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <property name="horizontalSpacing">
           <number>16</number>
          </property>
          <item row="2" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Übersicht über die Vermietungungen.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;- Welche Marke wird am häufigsten vermietet?&lt;/span&gt;&lt;/p&gt;&lt;p&gt;- Welches Modell ist gefragtesten?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string># Statistiken</string>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::MarkdownText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="BarChart" name="barChart" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>300</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PieChart" name="pieChart" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 127);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="BarChart" name="barChart_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarChart</class>
   <extends>QWidget</extends>
   <header>app.widgets.BarChart</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PieChart</class>
   <extends>QWidget</extends>
   <header>app.widgets.DonutCharts.PieChart</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
